dotnet/src/Connectors/Connectors.AzureOpenAI/Core/ClientCore.ChatCompletion.cs

```diff
@@ -92,25 +92,25 @@ private record ToolCallingConfig(IList<ChatTool>? Tools, ChatToolChoice Choice,
     private static Dictionary<string, object?> GetChatCompletionMetadata(OpenAIChatCompletion completions)
     {
 #pragma warning disable AOAI001 // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
-        return new Dictionary<string, object?>(8)
+        return new Dictionary<string, object?>
         {
             { nameof(completions.Id), completions.Id },
             { nameof(completions.CreatedAt), completions.CreatedAt },
-            { PromptFilterResultsMetadataKey, completions.GetContentFilterResultForPrompt() },
+            { ContentFilterResultForPromptKey, completions.GetContentFilterResultForPrompt() },
```

Comment:
Should we use `nameof` and avoid const?

```suggestion
            { ContentFilterResultForPromptKey, completions.GetContentFilterResultForPrompt() },
```

Reply:
nameof(GetContentFilterResultForPrompt) will return "GetContentFilterResultForPrompt", do we want to have metadata key starting from Get?